docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,356 @@
+# Using node local storage on kubernetes
```

Comment:
I would capitalize it (except "on")

Reply:
Use https://capitalizemytitle.com/ :)

Reply:
I wouldn't, but if all agree (and looking quickly it seems that is done in another docs), so did it :)